src/code/z_actor.c

```diff
@@ -776,25 +777,25 @@ void Actor_Kill(Actor* actor) {
 }
 
 void Actor_InitPosRot(Actor* actor) {
-    actor->posRot = actor->initPosRot;
+    actor->world = actor->home;
 }
 
 void Actor_SetHeight(Actor* actor, f32 offset) {
-    actor->posRot2.pos.x = actor->posRot.pos.x;
-    actor->posRot2.pos.y = actor->posRot.pos.y + offset;
-    actor->posRot2.pos.z = actor->posRot.pos.z;
+    actor->focus.pos.x = actor->world.pos.x;
+    actor->focus.pos.y = actor->world.pos.y + offset;
+    actor->focus.pos.z = actor->world.pos.z;
 
-    actor->posRot2.rot.x = actor->posRot.rot.x;
-    actor->posRot2.rot.y = actor->posRot.rot.y;
-    actor->posRot2.rot.z = actor->posRot.rot.z;
+    actor->focus.rot.x = actor->world.rot.x;
+    actor->focus.rot.y = actor->world.rot.y;
+    actor->focus.rot.z = actor->world.rot.z;
 }
 
 void func_8002D5F4(Actor* actor) {
```

Comment:
These 2 functions could also be named, maybe `Actor_SetWorldToShapeRot` and vice versa